flash-messages/Gemfile

```diff
@@ -0,0 +1,20 @@
+source "https://rubygems.org"
+
+gem "sinatra"
+gem "data_mapper"
+gem "rack-flash3"
+
+group :development do
+  # We use SQLite3 on our local machines
+  gem "sqlite3"
+  gem "dm-sqlite-adapter"
+
+  gem "dotenv"
+  gem "rerun"
+end
+
+group :production do
+  # We use PostgreSQL on Heroku, not SQLite3
+  gem "pg"
+  gem "dm-postgres-adapter"
+end
```

Comment:
Though we could resolve that by making a top-level config.ru that mounts each application and has a shared gemfile and such.